src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2531,6 +2531,7 @@ because xbuild doesn't support framework reference assemblies.
     MultiDexMainDexListFile="$(_AndroidMainDexListFile)"
     CustomMainDexListFiles="@(MultiDexMainDexList)"
     ProguardInputJarFilter="$(_AndroidProguardInputJarFilter)"
+    ExtraArgs="$(MultiDexClassListExtraArgs)"
```

Comment:
This should be `$(AndroidMultiDexClassListExtraArgs)` (`Android` prefix), and as @jonathanpeppers mentioned, this should also be documented.